generators/server/templates/src/main/java/package/repository/_UserRepository.java

```diff
@@ -41,9 +39,11 @@
     Optional<User> findOneByLogin(String login);
     <%_ if (databaseType == 'sql') { _%>
 
-    @Query(value = "select distinct user from User user left join fetch user.authorities",
-        countQuery = "select count(user) from User user")
-    Page<User> findAllWithAuthorities(Pageable pageable);
+    @EntityGraph(attributePaths = "authorities")
+    User findOneWithAuthoritiesById(<%= pkType %> id);
+
+    @EntityGraph(attributePaths = "authorities")
+    Optional<User> findOneWithAuthoritiesByLogin(String login);
```

Comment:
It's not flexible. Let's add one-to-many  `actions` to `user`. Sometimes it may be needed to find a user only with `authorities`, but sometimes only with `actions`. Now we have to create `findOneWithActionsById` and `findOneWithActionsByLogin`. Furthermore, one day we will need to find a user with `authorities` **and** `actions`. So `findOneWithAuthoritiesAndActionsById` and `findOneWithAuthoritiesAndActionsByLogin` will be created. I don't even speak about 3+ relations or even inner relations like `action.consequences`.